.bootkube_env

```diff
@@ -16,5 +16,6 @@ export NSERVER02='192.168.1.70'             ### THE PRIMARY DNS SERVER      ###
 export NSERVER03='8.8.8.8'                  ### THE SECONDARY DNS SERVER    ###
 export NSEARCH01='svc.cluster.local'        ### MODIFY FOR YOUR ENVIRONMENT ###
 export NSEARCH02='jinkit.com'               ### MODIFY FOR YOUR ENVIRONMENT ###
+export KUBEARCH='amd64'                     ### MODIFY FOR YOUR ARCHITECTURE###
```

Comment:
i have a search and replace nit request (which i'm going to fix in other areas myself actually). so the variable visually stands out a little more, can you change `KUBEARCH` to `KUBE_ARCH`? i need to do this on a couple others, and then i'll standardize on `KUBE_` for any variable prefix going forward just to make them stand out a bit more visually.